### PR TITLE
Registration page color contrast and fonts

### PR DESCRIPTION
**Pull request by foxbunny**
- Fixed contrast issue by overriding SUI (for now)
- Fix font styles for the page heading

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -95,6 +95,7 @@ Accessibility
 - Associate form labels with form controls in the registration form (:issue:`6059`, :issue:`6073`,
   :pr:`6076`, thanks :user:`foxbunny`)
 - Make dropdown menu fully accessible (:issue:`5896`, :pr:`5897`, thanks :user:`foxbunny`)
+- Improver registration form color contrast and font sizes (:pr:`6098`, thanks :user:`foxbunny`)
```
Comment (ThiefMaster) [inline CHANGES.rst:98]:
```suggestion
- Improve registration form color contrast and font sizes (:pr:`6098`, thanks :user:`foxbunny`)
```

**Comment by ThiefMaster** [comment]
image

The PDF download button on the "Programme" page looks a bit strange now.

**Comment by tomasr8** [comment]
Can we keep the original label color and just change the font color? Firefox doesn't seem to mind in terms of contrast..
(for reference current master/pr):
image
image

File: indico/web/views.py [hunk 2]
```diff
@@ -358,7 +358,7 @@ def _get_footer(self):
 
     def _apply_decoration(self, body):
         breadcrumbs = self._get_breadcrumbs()
-        return '<div class="header">{}</div>\n<div class="main">{}<div>{}</div></div>\n{}'.format(
+        return '<div class="header">{}</div>\n<main class="main">{}<div>{}</div></div>\n{}'.format(
```
Comment (tomasr8) [inline indico/web/views.py:361]:
```suggestion
        return '<div class="header">{}</div>\n<main class="main">{}<div>{}</div></main>\n{}'.format(
```

**Comment by foxbunny** [comment]
> Firefox doesn't seem to mind in terms of contrast

What do you mean?

**Comment by foxbunny** [comment]
Here's what the contrast checker says:

image

**Comment by tomasr8** [comment]
> > Firefox doesn't seem to mind in terms of contrast
> 
> What do you mean?

I meant just setting `color: #333` w/o changing the background color

**Comment by foxbunny** [comment]
Ah ok, yeah, that also works.

**Comment by ThiefMaster** [comment]
Found another small side-effect that you probably did not intend (and it doesn't look good ;))

image
image

**Comment by foxbunny** [comment]
I'll take a look tomorrow.

**Comment by foxbunny** [comment]
> Found another small side-effect that you probably did not intend

I was on the wrong branch when I was fixing this, so the fix is in the #5945.

On a separate note, the font-family declaration for the Roboto fonts is incorrect. Roboto Light is declared as a font family on its own, but it's part of the Roboto family, and "Light" refers to its `font-weight`. The result is that the Light font doesn't apply when the font-weight is set to bold.

The ideal approach would be to declare the font family correctly, set all headings to Roboto, and vary only their weight as appropriate (e.g., smaller font-sizes should use Regular rather than Light). That should bring a lot more consistency to heading styling.

**Comment by ThiefMaster** [comment]
Any chance you could move the fix in here? I would really prefer to merge this PR - which is already fully reviewed - without having broken display (on some of our smaller instances we have I deploy from `a11y` rebased to the latest master to see if anyone encounters issue when using Indico regularly as a user)

**Comment by foxbunny** [comment]
Sure, let me take care of that.

**Comment by foxbunny** [comment]
Done

**Comment by ThiefMaster** [comment]
hmmm I don't really see a difference?

image

FWIW the size of "Settings" is nice, but the large text inside the dropdown there looks somewhat out of place

**Comment by foxbunny** [comment]
> hmmm I don't really see a difference?

In that case I have no idea what you were referring to to begin with :)

**Comment by ThiefMaster** [comment]
does the text  of the h2 ("Contributions") and in the dropdown menu not look too big to you?

**Comment by foxbunny** [comment]
> does the text of the h2 ("Contributions") and in the dropdown menu not look too big to you?

You do realize that's quite subjective? 😂

**Comment by foxbunny** [comment]
I'll update that tomorrow. Will also convert that dropdown since we're dealing with it.

**Comment by ThiefMaster** [comment]
Thanks!  I was actually about to ask whether using your new dropdown for this would make sense there :)

PS: H1 and H2 having pretty much the same size being strange is "subjective"? ;)

**Comment by foxbunny** [comment]
> PS: H1 and H2 having pretty much the same size being strange is "subjective"? ;)

Well, yes, because they're different colors. I don't really mind it being about the same size. Easier to read for me, too.

**Comment by foxbunny** [comment]
Speaking of H1, it's actually not an H1, which is a separate issue. Might as well fix that, too, since we're touching management areas anyway.

**Comment by foxbunny** [comment]
I've changed the font size of the H2. I didn't convert the dropdowns because they aren't merged yet (stuck in #5945). I also didn't change the title to H1 because H1 has some default styles and dealing with them would be a bit off-topic for this PR.

**Comment by ThiefMaster** [comment]
Just a general thought, maybe it'd be good to add new components and custom elements in separate commits that can be easily cherry-picked. As you probably realized I generally dislike merging PRs when things still look broken, and in cases like this one it could be easily solved by cherry-picking the component into the PR that needs it.

**Comment by foxbunny** [comment]
Would that not just keep expanding the scope of each PR? I mean, I don't mind bunkering down for weeks working on a big PR, but I'm trying to cut the scope down because it seemed you preferred it that way.

**Comment by ThiefMaster** [comment]
Indeed, I prefer smaller PRs over huge ones. But I also prefer not to have "broken" looks. Damn.. :D

Then again this is why I created the a11y branch so maybe I should not really worry about very slightly broken things especially as long as they will eventually be fixed in another PR... ;)

So, I'll merge this one :)

**Comment by ThiefMaster** [comment]
FYI I'll also rebase `a11y` to the latest `master` since you'll have to rebase your other PRs anyway.

**Comment by foxbunny** [comment]
> very slightly broken things

Btw, I did fix the dropdown layout, so it's slightly more or less the same way it was, only slightly bigger. :)

**Comment by foxbunny** [comment]
> FYI I'll also rebase a11y to the latest master since you'll have to rebase your other PRs anyway.

Thanks.

**Pull request merged**